### ISSUE TITLE
[meta] Fix redirection issues on E10s

### ISSUE THREAD

**Issue by myrdd** [RequestPolicyContinued/requestpolicy#721]
When E10s is enabled, the _redirection notification bar_ (r21n) has several issues. There are already Marionette unit tests for some of those issues [1], but however it's not trivial to solve those issues. Probably the r21n issues depend on #610.

[1] https://github.com/RequestPolicyContinued/requestpolicy/tree/dev-1.0/tests/marionette/tests/redirections

**Comment by myrdd** [comment]
wontfix. RP will be a webextension. --> #704
